asio/.cmake-format

```diff
@@ -0,0 +1,30 @@
+parse:
+  additional_commands:
+    foo:
+      flags:
+      - BAR
+      - BAZ
+      kwargs:
+        NAME: '1'
+        DEPENDS: '*'
+        HEADERS: '*'
+        SOURCES: '*'
+
+format:
+  dangle_parens: true
+  keyword_case: upper
+  line_ending: unix
+  line_width: 120
```

Comment:
I formatted all `cmake files` this this configuration!

**TODO**: `cmake-linit cmake/*.cmake`

Reply:
Sure, I don't mind. Personally I don't use this tool, because it results in weird formatting.